fakemenot/__init__.py

```diff
@@ -1,194 +0,0 @@
-        sys.exit(2)
-    return config
-
-
-def _do_ocr_and_lookup(img_obj):
-    config = get_config()
-    limit_of_tweets = int(args.limit)
-    potential_user = '__fakemenot__'
-    # Replace line breaks with a space and split text into an array
-    text = pytesseract.image_to_string(
-        img_obj, lang='eng').replace(
-        '\n', ' ').split(' ')
-    for element in text:
-        if element and element[0] == '@':
-            print("Detected handle : " + str(element))
-            # Since handles cannot have spaces, strip until space
-            potential_user = element.split(' ')[0]
-            break
-
-    # Just in case the person Yousing the program puts in ' or " in the config.
-    consumer_key = config.get(
-        'twitter',
-        'consumer_key').replace(
-            '\'','').replace(
-                '\"','')
-    consumer_secret = config.get(
-        'twitter',
-        'consumer_secret').replace(
-            '\'','').replace(
-                '\"','')
-    access_token = config.get(
-        'twitter',
-        'access_token').replace(
-            '\'','').replace(
-                '\"','')
-    access_token_secret = config.get(
-        'twitter',
-        'access_token_secret').replace(
-            '\'','').replace(
-                '\"','')
-
-    if potential_user == '__fakemenot__':
-        print(colored("[*] It looks like OCR failed. Please make sure you " +
-            "crop the image as in sample and is readable.", 'red'))
-        exit(1)
-
-    try:
-        tuo = TwitterUserOrder(potential_user[1:])
-        ts = TwitterSearch(
-            consumer_key=consumer_key,
-            consumer_secret=consumer_secret,
-            access_token=access_token,
-            access_token_secret=access_token_secret
-        )
-        tweets = []
-        body = '__awesomebody__'
-        for tweet in ts.search_tweets_iterable(tuo):
-            # Nobody cares about re-tweets
-            if 'RT ' not in tweet['text']:
-                if tweet not in tweets:
-                    tweets.append((tweet['text'], tweet['id']))
-                if not limit_of_tweets:
-                    break
-                else:
-                    limit_of_tweets -= 1
-
-        # The most probable tweet body is this.
-        try:
-            body = text[text.index('V') + 1:]
-        except ValueError:
-            body = text
-
-        # If none of that was found, let's report an OCR error
-        if body == '__awesomebody__':
-            print(colored("[*] It looks like OCR failed.Please make sure you " +
-                          "crop image as in sample and is readable.", 'red'))
-
-        found_tweet = False
-        # Check against every tweet pulled
-        for tweet in tweets:
-            removed_elements = 0
-            ltweet, orig_len = tweet[0].split(' '), len(tweet[0].split(' '))
-            # Compare each element of body to element in body. TODO: Optimize
-            for ele in body:
```

Comment:
Body is being passed as a string here. So, every iteration is going to produce a single character. It's best to `body.split(' ')`  before this :)

Reply:
I didnt touch that part. Just literally ripped out the OCR bits, e.g. the parts that set the variables potential_user and body.